nipoppy_cli/nipoppy/data/examples/sample_invocations/heudiconv-0.12.2-convert.json

```
@@ -1,5 +1,5 @@
 {
-    "dicom_dir_template": "[[NIPOPPY_DPATH_SOURCEDATA]]/[[NIPOPPY_PARTICIPANT_ID]]/[[NIPOPPY_BIDS_SESSION]]/*",
+    "dicom_dir_template": "[[NIPOPPY_DPATH_SOURCEDATA]]/sub-{subject}/[[NIPOPPY_BIDS_SESSION]]/*",
```

```suggestion
    "dicom_dir_template": "[[NIPOPPY_DPATH_SOURCEDATA]]/{subject}/[[NIPOPPY_BIDS_SESSION]]/*",
```

- No BIDS prefixes in `sourcedata` subdirectories
- Is there a reason we don't want to use the `{session}` wildcard too?

---

>No BIDS prefixes in sourcedata subdirectories
No change needed since we do have BIDS prefixes in `sourcedata`. In fact this adds duplicate `ses` prefix in the hidden `.heudiconv` dir after `prepare` step. Shouldn't be a problem during actual conversion. 
> Is there a reason we don't want to use the {session} wildcard too?
Added and tested!